.githooks/pre-commit

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Fail script if any command fails to execute
+set -e
```

Comment:
You can usually achieve the same by passing the argument to the interpreter with `#!/bin/bash -e`.